### DISCUSSION TITLE
Expo expects version 50.0.13, even though the latest is 50.0.11

### DISCUSSION THREAD

**Discussion by welschmoor** [expo/expo#27651] [Archived: Questions] [answered]
How come?
![Screenshot 2024-03-14 at 09 30 45](https://github.com/expo/expo/assets/87126711/dd6ceba5-15c8-4aec-951a-b0644a2ec932)

**Comment by brentvatne** [comment, marked as answer]
hey there! the authoritative value here lives on https://exp.host/--/api/v2/versions -- that said, i went ahead and updated expo@50.0.13 to latest on npm as well
